fw/http.c

```diff
@@ -3029,14 +3029,49 @@ tfw_h1_rewrite_purge_to_get(struct sk_buff *head)
 	return -ENOMEM;
 }
 
+/**
+ * Removes n bytes from the beginning of the message, iterating over skb chain if needed.
+ * @return zero on success and error code on error.
+ *
+ * TODO: to be replaced with future version of ss_skb_chop_head_tail() over all skbs of the message
+ */
+static int
+tfw_h1_chop_leading_crlf(struct sk_buff **head, unsigned n)
+{
+	struct sk_buff *skb = *head;
```

Comment:
This function can be simplified if you move the line into the loop. In this way the last line `skb = *head;` can be removed.

Reply:
True